docs/examples/UWyoming_UFS-CHEM_Pairing.ipynb

```diff
@@ -0,0 +1,228 @@
+{
+ "cells": [
```

Comment:
All these examples look great. Can you also add your examples to this list: https://github.com/L12D2/MELODIES-MONET/blob/develop/docs/examples/intro_examples.rst